releases/release-1.28/release-notes/maps/pr-117656-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 117656
+releasenote:
+  text: Remove GAed feature gate DevicePlugins.
```

Comment:
```suggestion
  text: Removed (GA) feature gate for: `DevicePlugins`.
```

Reply:
I also disliked the "GAed" expression, but it's the one [used in the official documentation](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates-removed/) (although with an apostrophe).

What do you think about something like:

"Removed GA'ed feature gate: "

Reply:
that would work! I like it, I hadn't checked how the docs had it

Reply:
Done, added changes and you as co-author the commit.